### ISSUE TITLE
Arrivals airlocks do not opem immediately, but instead after 15 seconds

### ISSUE THREAD

**Issue by ChocolateBickie** [tgstation/tgstation#67140]
## Round ID:
Multiple rounds (needs confirming)

## Reproduction:
Unpower arrivals. Touch airlocks to immediately open them, and watch with horror as the airlock takes 15 seconds to open.

This is worse when it comes to the second inner airlock as the shuttle will leave once you leave the shuttle, leaving you exposed to the vacuum of space.

Even worse, the pressure difference will drag you from the airlock, forcing you to repeat the process, inevitably resulting in death.

Comments by Orange indicate this is not how they are supposed to work. 

I think (and please forgive me if I am wrong because I am not a coder) the issue can be found here;
https://github.com/tgstation/tgstation/blob/14beb91cc5fcabf80781534b0624e2dac2a989d8/code/game/machinery/doors/airlock_types.dm

line 367: ''if(do_after(user, 15 SECONDS, target = src))''

If possible could this value be lowered or removed?

**Comment by optimumtact** [comment]
how I intended them to work, just to clarify is that even when unpowered they simply act like they are powered

**Comment by msgerbs** [comment]
@dragomagol I'm not sure why this is "not a bug", it sounds like this is not working how Orange intended. Reading the code, it seems like it was *intended* for the 15 second delay to occur from the *inside* of the airlock, instead it's applying any time the airlock is unpowered, which makes no sense.

How I understand the intended design:
Inside: If user is not authorized and door is powered: Wait 15 seconds for safety unlock. If not powered, need a crowbar.
Outside: Always open immediately whether powered or not.

**Comment by optimumtact** [comment]
yes you are right, it's definitely a bug, I wanted people to be able to get into the station and at least into the airlock when there was no power.
